08-communicating-results.Rmd

```diff
@@ -35,36 +37,34 @@ This sentence provides a few key pieces of information for the audience:
 
-It is important to remember that how we, as researchers, discuss these findings can greatly impact how the audience interprets the findings. There, we should take extreme caution when talking about and presenting results.
+It is important to remember that how we, as researchers, discuss these findings can greatly impact how the audience interprets the findings. Therefore, we should take extreme caution when talking about and presenting results.
 
 ## Visualizing Data
 
-Including data tables and graphs are used to portray a large amount of data in a concise manner. Although discussing key findings in the text is important, it is easier for the audience to digest large amounts of data in graphical or table format. When used correctly, combining text, tables, and graphs is extremely powerful in presenting results. This section provides examples of using the {gtsummary} and {ggplot} packages to enhance the dissemination of results.
+Including data tables and graphs are used to portray a large amount of data in a concise manner. Although discussing key findings in the text is important, it is often easier for the audience to digest large amounts of data in graphical or table format. When used correctly, combining text, tables, and graphs is extremely powerful in presenting results. This section provides examples of using the {gtsummary} and {ggplot} packages to enhance the dissemination of results.
 
 ### Tables
 
-Tables are a great way to provide a large amount of data when we want the individual data points to be read. However, it is important to present tables in a readable format. Numbers should be aligned, and rows and columns should be easy to follow. Using key visualization techniques, we can create tables that are informative and nice to look at. Many packages can be used to create easy-to-read tables (e.g., {kable} + {kableExtra}, {gt}, {gtsummary}, {DT}, {formattable}, {flextable}, {reactable}). We will focus on {gt} and {gtsummary} here, but we encourage learning about others as they may have additional helpful components.
+Tables are a great way to provide a large amount of data when we want the individual data points to be read. However, it is important to present tables in a readable format. Numbers should be aligned, and rows and columns should be easy to follow. Using key visualization techniques, we can create tables that are informative and nice to look at. Many packages can be used to create easy-to-read tables (e.g., {kable} + {kableExtra}, {gt}, {gtsummary}, {DT}, {formattable}, {flextable}, {reactable}). We will focus on {gt} and {gtsummary} here, but we encourage learning about others as they may have additional helpful components. 
```

Comment:
How much do you want this section to explain how to use gt? We don't mention the `gt()` function and what it does. We could also explain line-by-line how gt constructs a table (`tab_spanner`, `cols_label`).